src/radical/pilot/agent/agent_0.py

```diff
@@ -6,11 +6,13 @@
 import os
 import pprint
 import stat
+import threading
 import time
+from concurrent.futures import ThreadPoolExecutor
 
 import radical.utils       as ru
 
-from ..   import utils     as rpu
+from .. import utils as rpu, session
```

Comment:
This syntax is confusing.  Do you need to import `..session`?  I don't see it being used.  If you need it, please import on a separate line.

Reply:
removed. not used